token/core/zkatdlog/crypto/sigproof/membership.go

```diff
@@ -51,8 +51,6 @@ type MembershipWitness struct {
 	value *math.Zr
 	// hash is hash of value and it is also signed with Pointcheval-Sandes signature
 	hash *math.Zr
-	// sigBlindingFactor is the randomness used to obfuscate signature
-	sigBlindingFactor *math.Zr
```

Comment:
Kaoutar, please, can you check this?

Reply:
Let's mark this as unused and add a TODO for @KElkhiyaoui to check when she is back from vacation, so this does not block the entire PR

Reply:
cc @alexandrosfilios